package.json

```diff
@@ -201,7 +201,7 @@
         "ilib-tree-node": "^1.2.0",
         "immutable": "^3.7.4",
         "intl": "^1.2.5",
-        "intl-messageformat": "^2.2.0",
+        "intl-messageformat": "^7.2.1",
```

Comment:
Probably want to revert this line. I saw that this version was throwing jest errors previously, which is why I left it at 2.2.0.

Reply:
will do